backend/src/ee/routes/v1/project-role-router.ts

```diff
@@ -32,18 +33,11 @@ export const registerProjectRoleRouter = async (server: FastifyZodProvider) => {
         projectSlug: z.string().trim().describe(PROJECT_ROLE.CREATE.projectSlug)
       }),
       body: z.object({
-        slug: z
-          .string()
-          .toLowerCase()
-          .trim()
-          .min(1)
+        slug: slugSchema()
```

Comment:
hmmm we've removed the `.toLowerCase` behavior here and it's not in the slugSchema